modules/toxiproxy/src/main/java/org/testcontainers/containers/ToxiproxyContainer.java

```diff
@@ -21,9 +21,10 @@
  */
 public class ToxiproxyContainer extends GenericContainer<ToxiproxyContainer> {
 
+    public static final int TOXIPROXY_CONTROL_PORT = 8474;
```

Comment:
could you please revert this change and add `getControlPort` instead? We find it more flexible than exposing constants

Reply:
Sure, do you want to expose the original or the mapped port?

Reply:
the mapped one, so that it can be used as an argument to the client